### PR TITLE
Add CakeYaml plugin

### PR DESCRIPTION
**Pull request by makallio85**
https://github.com/makallio85/cake-yaml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,6 +176,7 @@ Additional lists you might find useful:
 *Useful libraries or tools that don't fit in any of the other categories.*
 
 - [Aura.Intl](https://github.com/auraphp/Aura.Intl) - A powerful I18n library - used in CakePHP3.x core.
+- [Cake-yaml](https://github.com/makallio85/cake-yaml) - Configure CakePHP 3 routes with simple yaml files.
```
Comment (ADmad) [inline README.md]: "YAML" instead of "yaml" would be better since it's an acronym.

**Comment by ADmad** [comment]
I would prefer if the plugin had tests before it got added to the list.

**Comment by makallio85** [comment]
I added integration test, seems to be running just fine.

**Comment by dereuromark** [comment]
This is a plugin? It should follow the contribution guidelines!
The name is "CakeYaml plugin" then. 
Also the description could be improved. No need to mention CakePHP or the version number as both are clear from context, so maybe just

     Configure routes with simple YAML files.

**Comment by dereuromark** [comment]
It would also better fit in the plugins section then: https://github.com/FriendsOfCake/awesome-cakephp#miscellaneous

**Comment by makallio85** [comment]
Okay, good points. Just one thing. This is actually library and plugin. But not just one. It works only with CakePHP 3 but does not work like a plugin.

So is it plugin or not?

**Comment by dereuromark** [comment]
What do you mean it does not work like a plugin?
You are activating it by using Plugin::load(), are you not? That classifies as plugin usage for me :)

PS: I also made the PR message a little bit less vague.

**Comment by makallio85** [comment]
Okay, then it is not plugin. It is not loaded like that.

**Comment by makallio85** [comment]
Then I'll change:
- description to "Configure routes with simple YAML files."
- package name to yaml-route

Thats is?

**Comment by dereuromark** [comment]
Ah I see, ok sonds good :)

**Comment by makallio85** [comment]
I made those changes. It should fine now.

File: README.md [hunk 2]
```diff
@@ -176,6 +176,7 @@ Additional lists you might find useful:
 *Useful libraries or tools that don't fit in any of the other categories.*
 
 - [Aura.Intl](https://github.com/auraphp/Aura.Intl) - A powerful I18n library - used in CakePHP3.x core.
+- [Yaml-route](https://github.com/makallio85/yaml-route) - Configure routes with simple YAML files.
```
Comment (dereuromark) [inline README.md]: YamlRoute?

**Comment by makallio85** [comment]
YamlRoute it is. Ordered list with update also.

**Comment by dereuromark** [comment]
Cool, thx!

**Pull request merged**